src/mutations/itemTag.ts

```diff
@@ -65,12 +67,14 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
       });
     },
     onError: (error, { id }, context) => {
-      const itemKey = buildItemTagsKey(id);
+      const itemKey = itemTagsKeys.singleId(id);
       queryClient.setQueryData(itemKey, context.itemTags);
       notifier?.({ type: deleteItemTagRoutine.FAILURE, payload: { error } });
     },
     onSettled: (_data, _error, { id }) => {
-      queryClient.invalidateQueries(buildItemTagsKey(id));
+      queryClient.invalidateQueries(itemTagsKeys.singleId(id));
+      // invalidate any "many" query that contains the id we modified
+      queryClient.invalidateQueries(itemTagsKeys.manyIds([id]));
```

Comment:
Would it make sense to have a "util" of some sort to handle an invalidation of single and many given an id?  Not sure how

Reply:
I guess we could make a util that would invalidate single and many queries relating to a given id, not sure of the benefit though...